packages/ui/src/working-groups/hooks/useOpeningsPagination.ts

```diff
@@ -27,7 +27,7 @@ export const useOpeningsPagination = ({ groupId, type, page = 1 }: UseOpeningsPa
     limit: OPENINGS_PER_PAGE,
     offset: (page - 1) * OPENINGS_PER_PAGE,
     where,
-    order: [WorkingGroupOpeningOrderByInput.CreatedAtDesc, WorkingGroupOpeningOrderByInput.RuntimeIdDesc],
+    order: WorkingGroupOpeningOrderByInput.CreatedAtDesc,
```

Comment:
So this change should be:
```suggestion
    order: [WorkingGroupOpeningOrderByInput.CreatedAtDesc],
```
But I don't understand why does this solve the issue. Given that there is an issue to begin with: see my comment on [#2513](https://github.com/Joystream/pioneer/issues/2513#issuecomment-1437373412)

Reply:
The goal of issue https://github.com/Joystream/pioneer/issues/2513 is to sort openings according to "created date", not by "end date".
It's true that created date is displayed nowhere in the UI, but that's irrelevant to the issue. Maybe we can add "created date" of openings as a new column.